docs/contribute/plugin/develop_plugin_rustsdk.md

```diff
@@ -1,36 +1,79 @@
+---
+sidebar_position: 4
+---
+
 # Develop WasmEdge Plug-in in Rust SDK with witc
 
-Once you complete C++ plugin code, you can use witc[^1] to generate Rust Plugin SDK
```

Comment:
This is an important step. Don't ignore this. The prerequisites are that we already have a C++ pugin code.